# try:
#     numerator = 10
#     denominator = 0

#     result = numerator/denominator
#     print(result)
# except:
#     print("error:denominator cannot be 0")

# try:
#     print("hello")
# except:
#     print("something went wrong")
# else:
#     print("nothing went wrong")

try:
    print(x)
except:
    print("something went wrong")
finally:
    print("the 'try except' is finished")
            